solidity/contracts/TokenStakingEscrow.sol

```diff
@@ -247,6 +276,19 @@ contract TokenStakingEscrow is Ownable {
         emit DepositWithdrawn(operator, grantee, amount);
     }
 
+    function withdrawRevoked(
+        Deposit memory deposit,
+        address operator,
+        address grantManager
+    ) internal {
+        uint256 amount = deposit.amount - deposit.withdrawn;
```

Comment:
Should we use SafeMath's `.sub`  here?

Reply:
Yeah, I think it's a good practice. We should never have `depositAmount > deposit.withdraw` but better be safe than sorry.